components/sidebar/sidebar_channel/sidebar_channel_link/sidebar_channel_link.tsx

```diff
@@ -253,23 +249,22 @@ export default class SidebarChannelLink extends React.PureComponent<Props, State
                     unreadMentions={unreadMentions}
                     hasUrgent={hasUrgent}
                 />
-                <SidebarChannelMenu
-                    channel={channel}
-                    channelLink={link}
-                    isMenuOpen={this.state.isMenuOpen}
-                    isCollapsed={this.props.isCollapsed}
-                    isUnread={isUnread}
-                    closeHandler={this.props.closeHandler}
-                    onToggleMenu={this.handleMenuToggle}
-                />
+                <div className='MenuWrapper SidebarMenu'>
```

Comment:
What's the menu wrapper needed for? Is it just because the trigger element needs to use some of the existing styles?

Reply:
yes i dont like it either its for existing styling to work. Let me see if i can tweak the styled a bit.

Reply:
removed MenuWrapper class